internal/civisibility/utils/git_test.go

```diff
@@ -586,3 +611,11 @@ func TestGetBaseBranchShaWithCIBaseBranch(t *testing.T) {
 		t.Logf("GetBaseBranchSha without CI tags returned valid SHA: %s", sha)
 	}
 }
+
+func acceptableError(err error) bool {
+	errMessage := err.Error()
+	// if the error is related to a shallow.lock file, we will skip the test;
+	// if the error is to a github connection error, we will skip the test;
+	// the test is flaky in the CI due to multiple git commands running at the same time.
+	return strings.Contains(errMessage, "shallow.lock") || strings.Contains(errMessage, "Couldn't connect to server")
```

Comment:
@tonyredondo Can we do a case-insensitive comparison? Just in case they change the `Could` by `could`. Who knows what can happen in the future.

Reply:
done here [case insensitive comparison](https://github.com/DataDog/dd-trace-go/pull/4297/commits/782a80967acbe230f437bdbc142ebd08024688dd)